packages/sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@nmshd/connector-sdk",
-    "version": "2.2.4",
+    "version": "2.3.0",
```

Comment:
3.0.0 (breaking change)

Reply:
@slavistan you resolved but not changed this.

Reply:
please don't forget to change it in the changelog too

Reply:
Oh, my bad. I misread `3.0.0` as `2.3.0`.